src/dictionaries/hu/news.json

```
@@ -21,7 +21,15 @@
   "rendezvenyek": {
     "title": "Rendezvények",
     "description": "Az Egyetemi Hallgatói Képviselet és a hallgatói szervezetek által szervezett események és programok.",
```

Biome (2.5.0)

[error] The key description was already declared.

(lint/suspicious/noDuplicateObjectKeys)